frontend/src/uxbox/util/i18n.cljs

```diff
@@ -16,7 +16,7 @@
    [uxbox.util.transit :as t]
    [uxbox.util.storage :refer [storage]]))
 
-(defonce locale (get storage ::locale cfg/default-language))
+(defonce locale (get storage ::locale cfg/default-lang))
```

Comment:
Just a detail but, why rename this field ?
It might be better if we keep PRs strictly related to one feature, and renaming language fields are not related to theme management, right ?

Reply:
Yeah. I just can not pass by an inconsistent lang and language.